tools/sense_studio/static/main.js

```diff
@@ -161,6 +161,22 @@ function syncRequest(url, data) {
 }
 
 
+function sendGetRequest(url, data) {
+    let xhttp = new XMLHttpRequest();
+
+    xhttp.open('GET', url, false);
+    xhttp.setRequestHeader('Content-type', 'application/json; charset=utf-8');
+
+    if (data) {
+        xhttp.send(JSON.stringify(data));
+    } else {
+        xhttp.send();
+    }
+
+    return JSON.parse(xhttp.responseText);
+}
+
+
```

Comment:
Try to combine `sendGetRequest` and `syncRequest` into one method, What you can do here is pass one more parameter as function argument `method`, whose value will be `'GET'` or `'POST'` accordingly.

where for `xhttp.setRequestHeader('Content-type', 'application/json; charset=utf-8');` you can use `method` as condition, because it is only needed to be sent when request type is `POST`.

Reply:
Change accordingly in the code wherever `syncRequest` is used with appropriate method.

Reply:
Add some description in the description box of PR.